com.kgurgul.cpuinfo.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.kgurgul.cpuinfo
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: cpuinfo
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=host
```

Comment:
why does it need `host` filesystem access?

Reply:
Not needed - removed